.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -60,3 +59,16 @@ Uncomment and fill if you use not casual environment or if it might be relevant.
 - Others:
 
 -->
+
+**Screenshots**
```

Comment:
hm, all of this is already present in the comment. Do you mean we should uncomment what we have above? (:

Reply:
The contribution guide should be present on the issue template the main goal behind this is new contributor can easily set-up on their local system. Let's assume the issue is related to website so in the website template their should be a guidelines on how to setup website on their local system :)